Documentation/DebugApi.md

```diff
@@ -22,11 +22,11 @@ from pprint import pprint
 
 # create an instance of the API class
 api_instance = amun.swagger_client.DebugApi()
-specification = amun.swagger_client.InspectionSpecification() # InspectionSpecification | Specification of the software stack that should be created and verified.
+inspection_specification = amun.swagger_client.InspectionSpecification() # InspectionSpecification | Specification of the software stack that should be created and verified.
 
 try:
     # Generate Dockerfile as it would be generated internally for inspections. 
-    api_response = api_instance.post_generate_dockerfile(specification)
+    api_response = api_instance.post_generate_dockerfile(inspection_specification)
```

Comment:
The issue has been resolved! Thanks to @pacospace!!! :clap: :clap:

Reply:
Let's use new Amun finally :smiley: